frontend/public/components/factory/table.tsx

```diff
@@ -104,12 +105,38 @@ const sorts = {
   serviceCatalogStatus,
   jobCompletions: (job) => getJobTypeAndCompletions(job).completions,
   jobType: (job) => getJobTypeAndCompletions(job).type,
+  managedNamespace: (obj) => {
+    const olmOperatorGroup = _.get(obj, 'metadata.annotations["olm.operatorGroup"]');
+    let olmManagedNamespace;
+    if (
+      !_.isUndefined(olmOperatorGroup) &&
+      olmOperatorGroup !== null &&
+      olmOperatorGroup === 'global-operators'
+    ) {
+      olmManagedNamespace = 'All Namespaces';
+    } else {
+      const namespaces = _.get(obj, 'metadata.annotations["olm.targetNamespaces"]');
+      const namespacesList = _.isUndefined(namespaces) ? [] : namespaces.split(',');
+      namespacesList.length > 1
+        ? (olmManagedNamespace = `{namespacesList.length} Namespaces`)
+        : (olmManagedNamespace = namespacesList);
```

Comment:
Missing template `$` ?
```suggestion
      olmManagedNamespace = namespacesList.length > 1 ?`${namespacesList.length} Namespaces` : namespacesList;
```